bin/shodan

```diff
@@ -1070,6 +1077,13 @@ def stream(color, fields, separator, limit, datadir, ports, quiet, timeout, stre
 
     if datadir:
         fout = open_file(datadir, last_time)
+        fout.write('[\n')
```

Comment:
Yeah, I was afraid that this one would be hard ingest, since it breaks the existing format. However, unfortunately most (even streaming!) JSON libs such as yajl2 do require [ ] array markers. 
So, I am not quite sure what's the best approach.

Probably to leave the format as it is and add a flag --strict-json ?